CHANGELOG.md

```diff
@@ -30,6 +30,8 @@
 * Update LUX to activate when usage cookies are accepted ([PR #2154](https://github.com/alphagov/govuk_publishing_components/pull/2154)) PATCH
 * Add custom dimension to the page view data for pages tagged to brexit taxons. And remove the custom dimension from superbreadcrumb clicks. ([PR #2186](https://github.com/alphagov/govuk_publishing_components/pull/2186))
 
+* Add intervention component ([PR #2196](https://github.com/alphagov/govuk_publishing_components/pull/2196))
```

Comment:
This should be under the 'Unreleased' heading, after line 10.

Reply:
well spotted. Stupid git (not you, just git)